staging/src/k8s.io/apiserver/pkg/storage/etcd3/lease_manager.go

```diff
@@ -42,8 +42,8 @@ type leaseManager struct {
 }
 
 // newDefaultLeaseManager creates a new lease manager using default setting.
-func newDefaultLeaseManager(client *clientv3.Client) *leaseManager {
-	return newLeaseManager(client, 60, 0.05)
+func newDefaultLeaseManager(client *clientv3.Client, leaseReuseDurationSeconds int64) *leaseManager {
+	return newLeaseManager(client, leaseReuseDurationSeconds, 0.05)
```

Comment:
We could change this to an option struct if we can config a max size of a lease further.

Reply:
Let's do that when we will be adding the next argument.